perses/rjmc/geometry.py

```diff
@@ -289,8 +304,17 @@ def _logp_propose(self, top_proposal, old_positions, beta, new_positions=None, d
             the logp of the proposal
         new_positions : simtk.unit.Quantity with shape (n_atoms, 3) with units compatible with nanometers
             The new positions (same as input if direction='reverse')
+        rjmc_info: list
```

Comment:
Would it be possible to convert this `list` into a `dict` or some other structured object? I just worry that we could easily mess up this complicated list-based data structure if we ever insert more information into the object.

A `dict` is probably the simplest way to go!